entry_types/scrolled/package/src/frontend/navigation/AppHeader.module.css

```diff
@@ -151,3 +151,11 @@
     height: 10px;
   }
 }
+
+@media (min-width: 1025px) {
+    .chapterList {
+      width: 55%;
+      position: absolute;
+      left: 23%;
+    }
+}
```

Comment:
Why is this needed?

Reply:
The chapter list covers the whole navigation bar. So, if a user skips links one time then he has to use keyboard to use the skip link again. 
So if a user clicks on navigation bar and hit tab then without this the first chapter list item is focused.
This also doesn't provide a considerable solution as it only makes the chapter list smaller so the user has some space between the chapter list and the logo to click and use the skip link.

Reply:
I removed this for now. Let me know if this makes sense and I will add it again or any other idea you have for that.

Reply:
Yeah, I think that's fine. The user can always return to the top of the tab order by focusing the location bar (Crtl+L etc.)